events/nickname_events/wc_nickname_events.txt

```diff
@@ -0,0 +1,142 @@
+﻿###################
+# Nickname Events
+###################
+
+namespace = WCNIC
+
+### the Old Murkeye
+WCNIC.1 = { # Nickname for when the character rule long
+	type = character_event
+	title = WCNIC.1.t
+	
+	desc = {
+		triggered_desc = {
+			trigger = { has_culture_group = culture_group:murloc_group }
+			desc = WCNIC.1.desc.m
+		}
+		triggered_desc = {
+			trigger = { has_culture_group = culture_group:human_group }
+			desc = WCNIC.1.desc.h
+		}
+	}
+	
+	theme = default
+	override_background = {
+		event_background = feast 
+	}
+	left_portrait = root
+	
+	trigger = {
+		is_ruler = yes
+		
+		NOR = {
+			has_character_flag = had_nickname_event
+			has_character_flag = denied_nickname_event_the_old
+		}
+		
+		OR = {
+			has_culture_group = culture_group:human_group
+			has_culture_group = culture_group:murloc_group
+		}
+		
+		has_any_nickname = no
+		NOT = {
+			primary_title = { recent_history = { years = 10 } }
+		}
+		age >= 45
+	}
+	
+	weight_multiplier = {
+		base = 1
+		modifier = {
+			factor = 1
+			age >= 55
+		}
+		modifier = {
+			factor = 1
+			age >= 65
+		}
+		modifier = {
+			factor = 1
+			NOT = {
+				primary_title = { recent_history = { years = 20 } }
+			}
+		}
+		modifier = {
+			factor = 2
+			NOT = {
+				primary_title = { recent_history = { years = 30 } }
+			}
+		}
+	}
+	
+	immediate = {
+		add_character_flag = {
+			flag = had_nickname_event
+		}
+	}
+	
+	option = { # the Old Murkeye
+		name = WCNIC.1.a.m
+		flavor = {
+			desc = WCNIC.1.tt.a.m
+		}
+		
+		trigger = {
+			has_culture_group = culture_group:murloc_group
+		}
+		
+		set_nickname_effect = { NICKNAME = nick_the_old_murkeye }
+		give_self_nickname_effect = yes
+		add_prestige = minor_prestige_gain
+		
+		ai_chance = {
+			base = 100
+		}
+	}
+	
+	option = { # the Old
+		name = WCNIC.1.a.h
+		
+		trigger = {
+			has_culture_group = culture_group:human_group
+		}
+		
+		set_nickname_effect = { NICKNAME = nick_the_old }
+		give_self_nickname_effect = yes
```

Comment:
This effect is a local one from `nickname_events.txt`. It doesn't work here.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/118680289-a9ef8900-b80f-11eb-827e-b32378b1e739.png)

</details>

To make it work, you should move the definition from `nickname_events.txt` to some file in `Warcraft-Guardians-of-Azeroth-2\common\scripted_effects`. Maybe a new one. Note: using the type, `scripted_effect`, won't be needed there.

Reply:
I'm not sure it's done, because i'm not sure I understand what you want, but maybe in 30aa0b943

Reply:
> I'm not sure it's done, because i'm not sure I understand what you want, but maybe in [30aa0b9](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/commit/30aa0b9435b4a5ea3860000021db585ff2eb294f)

Yes, that's right. But to finish it you have to delete another definition from here:
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/121761656-2c831400-cb42-11eb-8cc2-b98474248ca2.png)

</details>

Reply:
done in b19067462